clinica/utils/inputs.py

```diff
@@ -267,28 +267,52 @@ def find_sub_ses_pattern_path(
 
 
 def _check_information(information: Dict) -> None:
-    if not isinstance(information, dict):
-        raise TypeError("A dict must be provided for the argument 'information'")
-
-    if not all(elem in information for elem in ["pattern", "description"]):
-        raise ValueError(
-            "'information' must contain the keys 'pattern' and 'description'"
+    if not isinstance(information, (dict, list)):
```

Comment:
Could you update the function's signature to reflect that `information` can be a list or a dict?